python/ecl/summary/ecl_sum.py

```diff
@@ -1481,8 +1481,8 @@ def export_csv(self, filename, keys=None, date_format="%Y-%m-%d", sep=";"):
 
 
 
-    def resample(self, new_case_name, time_points):
-        new_case = self._resample(new_case_name, time_points)
+    def resample(self, new_case_name, time_points, lower_extrapolation=False, upper_extrapolation=False):
```

Comment:
I this one burns as heavily as I believe in the C-code if you try to extrapolate with all extrapolation deactivated, I think it would be good to detect this case and raise a Python exception instead. That is also testable ;)

Reply:
Currently testing only via jenkins (any suggestions how to execute these tests locally?). We can have a chat about this tomorrow.

Reply:
I don't really understand the question; but basically:
```
cmake .. -DBUILD_TESTS=ON -DENABLE_PYTHON=ON ...
make -j 8
ctest -j 8 ...
```

Reply:
It crashes on this function:  
```ecl_sum_vector_type * ecl_sum_vector = ecl_sum_vector_alloc(ecl_sum, true);```
The testing question related to this decorator: @statoil_test() (i.e., skipping all the tests)

Reply:
Update: interpolation works, just the starting time for the sampling interval was given by the input summary instead of the given time points.  Need to finish the test tomorrow and it'll be good to go.